db/schema.rb

```diff
@@ -69,6 +69,24 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "favorites", force: :cascade do |t|
```

Comment:
@Za-hell can we keep the "favorite" spelling across the code as it is the one you decided on for the schema? thanks!